batch-index/src/test/java/no/unit/nva/search/keybatch/GenerateKeyBatchesHandlerTest.java

```diff
@@ -59,9 +59,22 @@ void setUp() {
 
     @Test
     void shouldPersistS3KeysToBatchBucket() throws JsonProcessingException {
-        final var allFiles = putObjectsInInputBucket(SINGLE_BATCH_FILE_SIZE);
+        final var allFiles = putObjectsInInputBucket(SINGLE_BATCH_FILE_SIZE, DEFAULT_LOCATION);
```

Comment:
it used to be called s3location, if this is aws location, why set it in code?

Reply:
This is not a s3 bucket, but s3 path, which can have two values - `resources` and `import-candidates`.
In this way we can use this lambda to generate batches for two indexes, where `resources` is default index.
It is optional variable that can be set from input event :)